geniza/templates/nav.html

```diff
@@ -66,33 +68,21 @@
                         </a>
                     </li>
                     <li class="menu-label" role="none">
-                        <span>{{ about_label }}</span>
+                        <span>{{ about_page.localized.title }}</span>
                     </li>
                     <li class="menu-button" role="none">
                         <a href="#" role="button">
                             <span class="sr-only">{{ close_button }}</span>
                         </a>
                     </li>
-                    <li class="menu-item" role="none">
-                        {# Translators: label for link to 'Credits' page in main navigation #}
-                        <a href="/credits/" role="menuitem">{% translate "Credits" %}</a>
-                    </li>
-                    <li class="menu-item" role="none">
-                        {# Translators: label for link to 'How to Cite' page in main navigation #}
-                        <a href="/how-to-cite/" role="menuitem">{% translate "How to Cite" %}</a>
-                    </li>
-                    <li class="menu-item" role="none">
-                        {# Translators: label for link to 'Data Exports' page in main navigation #}
-                        <a href="/data-exports/" role="menuitem">{% translate "Data Exports" %}</a>
-                    </li>
-                    <li class="menu-item" role="none">
-                        {# Translators: label for link to 'Technical' page in main navigation #}
-                        <a href="/technical/" role="menuitem">{% translate "Technical" %}</a>
-                    </li>
-                    <li class="menu-item" role="none">
-                        {# Translators: label for link to 'FAQ' page in main navigation #}
-                        <a href="/faq/" role="menuitem">{% translate "FAQ" %}</a>
-                    </li>
+                    {% for child_page in about_page.localized.get_children.live.in_menu %}
+                        <li class="menu-item" role="none">
+                            {# Translators: label for link to 'Credits' page in main navigation #}
+                            <a href="/{{ about_page.localized.slug }}/{{ child_page.localized.slug }}/" role="menuitem">
+                                {{ child_page.localized.title }}
```

Comment:
We shouldn't be constructing the url like this.  Can't we use `page_url` instead?

Also, I think the translators comment is out of date now.

```suggestion
                            <a href="{% pageurl child_page.localized %}" role="menuitem">
                                {{ child_page.localized.title }}
```